web/src/components/DandiFooter.vue

```diff
@@ -17,9 +17,8 @@ export default {
       const gitRev = process.env.VUE_APP_GIT_REVISION;
       if (gitRev) {
         return `https://github.com/dandi/dandiarchive/commit/${gitRev}`;
-      } else {
-        return "https://github.com/dandi/dandiarchive";
       }
+      return 'https://github.com/dandi/dandiarchive';
```

Comment:
Weird; do you know why this was passing lint tests?

Reply:
Yeah I don't know how it was passing before, very strange. It does fail now if I try to revert that line back to double quotes though.